cdap-ui/app/services/constants.js

```diff
@@ -43,27 +43,27 @@ angular.module(PKG.name + '.services')
       hydrator: {
         appLabel: 'Hydrator Pipeline',
         studio: {
-          noConfigMessage: 'No configuration found for plugin',
-          syntaxConfigJsonError: 'Syntax error in configuration JSON for plugin',
-          semanticConfigJsonError: 'Semantic error in configuration JSON for plugin',
+          noConfigMessage: 'No configuration found for the plugin.',
+          syntaxConfigJsonError: 'Syntax error in the configuration JSON for the plugin.',
+          semanticConfigJsonError: 'Semantic error in the configuration JSON for the plugin.',
           oneSinkError: 'Please add at least one sink to your pipeline.',
           sourceRequiredFieldsError: 'Please provide required fields for the source.',
           sinkRequiredFieldsError: 'Please provide required fields for the sink.',
           transformRequiredFieldsError: 'Please provide required fields for the transform.',
-          genericMissingRequiredFieldsError: 'Missing required fields',
+          genericMissingRequiredFieldsError: 'Missing required fields.',
           oneSourceError: 'Pipelines can only have one source. Please remove any additional sources.',
           noSourceError: 'Please add a source to your pipeline',
           nameError: 'Please name your pipeline.',
-          nameValidationError: 'Pipeline names can only contain alphanumeric (\'a-z A-Z 0-9\') characters and underscores ( \'_\'). Please remove other characters.',
+          nameValidationError: 'Pipeline names can only contain alphanumeric (\'a-z A-Z 0-9\') and underscore ( \'_\') characters. Please remove any other characters.',
           sinkBranchNodeError: 'Please connect multiple sinks to the same node.',
           branchError: 'Please remove branched connections.',
           unconnectedNodesError: 'Please connect all nodes.',
           circularConnectionError: 'Please remove the circular connection in this pipeline.',
           endSinkError: 'Please end the pipeline connections in a sink.',
           parallelConnectionError: 'Please remove parallel connections in this pipeline.',
-          pluginDoesNotExist: 'Plugin does not exist: ',
-          unsavedPluginMessage1: 'There are un-saved changes for node: ',
-          unsavedPluginMessage2: '. Please save them before publishing the pipeline'
+          pluginDoesNotExist: 'This plugin does not exist: ',
+          unsavedPluginMessage1: 'There are un-saved changes for node \'',
+          unsavedPluginMessage2: '\'. Please save them before publishing the pipeline.'
```

Comment:
1. Unsaved rather than un-saved (?)
2. Maybe this should be a single constant: 'There are unsaved changes for node '{}'. Please save them before publishing the pipeline.' Probably not supported though since we're doing this here..